website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -246,7 +246,7 @@ Find more information about materialized view limitations in Redshift's [docs](h
 #### Changing materialization from "materialized_view" to "table" or "view"
 
 Swapping a materialized view to a table or view is not supported.
-You must manually drop the existing materialized view in the data warehouse prior to calling `dbt run`.
+You must manually drop the existing materialized view in the data warehouse before calling `dbt run`.
```

Comment:
Hiya @mirnawong1 

I'm probably doing something wrong but I couldn't find this section in Vercel. I did find the previous (red) sentence in the dbt docs.

Kind Regards